### PR TITLE
Update all deps of all apps

### PR DESCRIPTION
**Pull request by ytimocin**

### CODE REVIEW

File: samples/demo/client/src/routes/Request.tsx [hunk 1]
```diff
@@ -7,8 +7,6 @@ export default function Request() {
     const [body, setBody] = React.useState("");
     const [reloadCount, setReloadCount] = React.useState(0);
 
-    const [title, setTitle] = React.useState("");
```
Comment (ytimocin) [inline samples/demo/client/src/routes/Request.tsx:10]: This was unused and it was throwing a warning.

File: samples/demo/client/src/routes/Request.tsx [hunk 2]
```diff
@@ -27,7 +25,7 @@ export default function Request() {
         return () => {
             mounted = false;
         }
-    }, [reloadCount])
+    }, [body, method, reloadCount, url])
```
Comment (ytimocin) [inline samples/demo/client/src/routes/Request.tsx:28]: This was also giving a warning.

File: samples/demo/src/db/mongo.ts [hunk 3]
```diff
@@ -44,8 +44,8 @@ export class MongoRepository implements Repository {
                 id: item.id,
                 title: item.title,
                 done: item.done,
-            })
-        return result.value as Item | null;
+            });
+        return result?.value as Item | null;
```
Comment (ytimocin) [inline samples/demo/src/db/mongo.ts:48]: This was throwing an error with the updated deps.

**Review by rynowak — COMMENTED** [review]
The code changes look reasonable. What did you do to test this out?

**Comment by ytimocin** [comment]
> The code changes look reasonable. What did you do to test this out?

I tested the demo locally and also one of the checks tests demo application with the updated **package.json** file.

**Review by rynowak — APPROVED** [review]

**Review by kachawla — APPROVED** [review]

**Pull request merged**